### PR TITLE
42 edit profile page

### PR DESCRIPTION
**Pull request by diana-k22**
I created a new profile edit page, added a back button and put breadcrumbs in a separate component

### CODE REVIEW

File: src/pages/editProfile/edit-profile.styled.js [hunk 1]
```diff
@@ -2,7 +2,6 @@ import styled from "styled-components";
 import {Button} from "../../globalStyled/styled.global";
```
Comment (Capsonchik) [inline src/pages/editProfile/edit-profile.styled.js:2]: codeStyle

File: src/App.js [hunk 2]
```diff
@@ -6,13 +6,15 @@ import { NotFoundPage } from './pages/notFound/not-found-page'
 import { ProfilePage } from './pages/profile/profile-page'
 import { Signin } from './pages/singnin/signin'
 import { Signup } from './pages/signup/signup'
+import {EditProfile} from "./pages/editProfile/edit-profile";
```
Comment (Capsonchik) [inline src/App.js]: codeStyle

File: src/components/breadcrumbs/breadcrumbs.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import {Link} from "react-router-dom";
+import useBreadcrumbs from "use-react-router-breadcrumbs";
+import styled from "styled-components";
```
Comment (Capsonchik) [inline src/components/breadcrumbs/breadcrumbs.jsx]: codeStyle

File: src/pages/editProfile/back-to-profile-btn/back-to-profile-btn.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import {Link} from "react-router-dom";
+import {ButtonBack} from "../edit-profile.styled";
+import styled from "styled-components";
```
Comment (Capsonchik) [inline src/pages/editProfile/back-to-profile-btn/back-to-profile-btn.jsx]: codeStyle

File: src/pages/editProfile/edit-profile.styled.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import {Button} from "../../globalStyled/styled.global";
+
+export const StyledEditProfile = styled.div`
```
Comment (Capsonchik) [inline src/pages/editProfile/edit-profile.styled.js]: Разве это не секция??

File: src/pages/editProfile/edit-profile.styled.js (hunk 1 above)
Comment (diana-k22) [reply src/pages/editProfile/edit-profile.styled.js:2]: +

**Pull request merged**